File: adminWF/Classes/UserChat.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace auctionComplex.Classes
{
    [Table("user_chats")]
    internal class UserChat
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("user_id")]
        public User User { get; set; }

        [Column("chat_id")]
        public AuctionChat AuctionChat { get; set; }
    }
}

File: serverSpring/src/main/java/com/vyatsu/auctionComplex/entity/api/UserChat.java
package com.vyatsu.auctionComplex.entity.api;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "user_chats")
public class UserChat {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    @Getter
    @Setter
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "\"UserId\"")
    @Getter
    @Setter
    private User user;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "\"AuctionChatId\"")
    @Getter
    @Setter
    private AuctionChat auctionChat;

}